src/Supplier/AddSupplierData/AddSupplierDataAddress.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Container,
+  Grid,
+  Input,
+  Textarea,
+  NumberInput,
+  HStack,
+  VStack,
+  Box,
+  PinInput,
+  PinInputField,
+} from '@chakra-ui/react';
+
+function AddSupplierDataAddress() {
+  return (
+    <form>
+      <VStack spacing={3} align="stretch">
+        <Box h="30" fontWeight="Bold">
+          Address
```

Comment:
Below you define labels in Polish, so it should be `Adres` if I'm not wrong.

Reply:
You have right, i have changed this